src/pages/CreateAssignment/CreateAssignment.js

```diff
@@ -61,15 +72,55 @@ function CreateAssignment({ firebase }) {
     setShowPhonics(false);
     setShowWriting(true);
   }
+  function verifyNameAndPush() {
+    if (
+      lessonName == null ||
+      lessonName == "" ||
+      typeof lessonName == "undefined"
```

Comment:
i think you might be able to do this check as `if (lessonName)` because `undefined`, `""`, and `null` are all falsey values in js but i'm not sure if the `typeof` check counts